dowhy/causal_estimator.py

```diff
@@ -641,10 +613,16 @@ def _evaluate_effect_strength(self, estimate, method="fraction-effect"):
         else:
             return None
 
-    def update_input(self, treatment_value, control_value, target_units):
-        self._control_value = control_value
-        self._treatment_value = treatment_value
-        self._target_units = target_units
+    def additional_evaluation(self, estimate: "CausalEstimate"):
```

Comment:
This is a forward reference, since CausalEstimate is defined below the CausalEstimator class we need to do use the quotes, otherwise we get a syntax error https://peps.python.org/pep-0484/#forward-references